src/Docfx.Dotnet/CompilationHelper.cs

```diff
@@ -102,12 +104,22 @@ public static Compilation CreateCompilationFromVBCode(string code, IDictionary<s
             references: GetDefaultMetadataReferences("VB").Concat(references ?? []));
     }
 
-    public static (Compilation, IAssemblySymbol) CreateCompilationFromAssembly(string assemblyPath, params MetadataReference[] references)
+    public static (Compilation, IAssemblySymbol) CreateCompilationFromAssembly(string assemblyPath, ExtractMetadataConfig? config = null, params MetadataReference[] references)
     {
+        var options = new CS.CSharpCompilationOptions(OutputKind.DynamicallyLinkedLibrary);
```

Comment:
It seems there is constructor that accept `metadataImportOptions` named parameter.
Is it able to use this overload instead of reflection?

Reply:
Good catch, I missed it!